apps/wing/README.md

```diff
@@ -1,42 +1,144 @@
+* [Native JSON](https://www.winglang.io/contributing/rfcs/language-spec#114-json-type) and schema validation support.
+* [Default immutability](https://www.winglang.io/blog/2023/02/02/good-cognitive-friction#immutable-by-default).
+* [Implicit async](https://www.winglang.io/contributing/rfcs/language-spec#113-asynchronous-model), explicit defer.
+
+For a more in-depth look at Wing's features and benefits, check out our [documentation](https://www.winglang.io/docs/).
+
+## Getting started 🛠️
+
+> 🚧 Wing is still in alpha, and not recommended for production use. But we're excited for anyone to take part in shaping our 
+> roadmap and contributing in any way. Our [project status](https://www.winglang.io/contributing/status) page includes more information about 
+> stability and roadmap.
+
+You can install Wing in a few simple steps:
+
+1. Check out the [Prerequisites](https://www.winglang.io/docs/start-here/installation#prerequisites).
+2. Install the [Wing CLI](https://www.winglang.io/docs/start-here/installation#wing-cli).
+3. Get the [Wing IDE Extension](https://www.winglang.io/docs/start-here/installation#wing-ide-extension) for your favorite editor.
+4. Launch the [Wing Console](https://www.winglang.io/docs/start-here/installation#wing-console) and take it for a spin!
+
+For a step-by-step guide, head over to our [Getting Started](https://www.winglang.io/docs/) guide.
+It's a once-in-a-lifetime adventure into the Wing rabbit hole!
+
+## FAQs ❓
+
+Here are some questions we're commonly asked that are covered by our [FAQ](https://www.winglang.io/docs/category/faq):
+
+* [Who is behind this project?](https://www.winglang.io/docs/faq/who-is-behind-wing)
+* [Which clouds are supported by Wing?](https://www.winglang.io/docs/faq/supported-clouds-services-and-engines/supported-clouds)
+* [Which provisioning engines are supported by Wing?](https://www.winglang.io/docs/faq/supported-clouds-services-and-engines/supported-provisioning-engines)
+
+## Community 💬
+
+Join our flock in the [Wing Slack](https://t.winglang.io/slack) community.
+We're here to help each other, answer questions, and share our cloud adventures.
+Alternatively, post any questions on [GitHub Discussions](https://github.com/winglang/wing/discussions).
+
+## Contributing 🤝
+
+Want to help Wing take flight?
+Check out our [contribution guide](https://github.com/winglang/wing/blob/main/CONTRIBUTING.md) to learn how to set up a development environment and contribute to the project.
+You can also get started by opening the project in GitHub Codespaces.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/winglang/wing)
+
+We are incredibly grateful to our entire community for contributing bug fixes and improvements:
+
+<a href="https://github.com/winglang/wing/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=winglang/wing" />
+</a>
+
+## License 📜
+
+Wing is licensed under the  [MIT License](./LICENSE.md).
```

Comment:
You're referencing the license in the cli dir here, which is different from our license on the root dir. Let's ask the expert if we should also copy the license.md here. @ShaiBer

Reply:
Discussed this with @ShaiBer and the conclusion is that it is actually the right approach to link to the license found on the same dir. Resolving.